api-reference/10 UI Components/dxDataGrid/3 Methods/totalCount().md

```diff
@@ -9,7 +9,128 @@ Gets the total row count.
 The total row count.
 
 ---
-[note]If any filter is applied, this method returns the count of records after [filtering](/concepts/05%20UI%20Components/DataGrid/30%20Filtering%20and%20Searching '/Documentation/Guide/UI_Components/DataGrid/Filtering_and_Searching/').
+Please review the following notes:
+
+- If any filter is applied, **totalCount()** returns the count of records after [filtering](/concepts/05%20UI%20Components/DataGrid/30%20Filtering%20and%20Searching '/Documentation/Guide/UI_Components/DataGrid/Filtering_and_Searching/').
+
+- If paging is disabled in the DataGrid ([paging](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/paging/).[enabled](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/paging/#enabled) is **false**) or in the underlying [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) ([paginate](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#paginate) is **false**), **totalCount()** returns -1. To get the total record count in this case, you can use the following code:
+
+    ---
+    ##### jQuery
+
+        <!-- tab: index.js -->
+        const dataGrid = $("#dataGridContainer").dxDataGrid("instance");
+        const DataSource = dataGrid.getDataSource();
```

Comment:
Why variable name is started from the letter in the upper case?

Reply:
I made this to make clear that it is our DataSource component, but since you ask, this is probably not such a good idea.